client/post-editor/media-modal/test/index.jsx

```diff
@@ -85,194 +76,229 @@ describe( 'EditorMediaModal', () => {
 	} );
 
 	test( 'When `single` selection screen chosen should initialise with no items selected', () => {
-		shallow( <EditorMediaModal { ...baseProps } single={ true } view={ null } /> ).instance();
+		const { container } = render(
+			<EditorMediaModal { ...baseProps } single={ true } view={ null } />
```

Comment:
We could use the opportunity to omit the redundant `={ true }`:
```suggestion
			<EditorMediaModal { ...baseProps } single view={ null } />
```